d2l-button-toggle-radio-group-item.js

```diff
@@ -0,0 +1,32 @@
+import {D2LButtonToggle} from './d2l-button-toggle.js';
+
+/**
+ * @customElement
+ * @polymer
+ */
+
+let subTemplate;
+class D2LButtonToggleRadioGroupItem extends D2LButtonToggle {
+	static get is() { return 'd2l-button-toggle-radio-group-item'; }
+
+	static get template() {
+		if (!subTemplate) {
+			subTemplate = super.template.cloneNode(true);
+			const button = subTemplate.content.querySelector('button');
+			button.setAttribute('aria-checked', '[[pressed]]');
+			button.setAttribute('aria-role', 'radio');
+			subTemplate.innerHTML = subTemplate.innerHTML.replace('aria-checked=', 'aria-checked$=');
```

Comment:
I don't love this approach of grabbing the `<button>` portion of the base class template and modifying it. I think it'll end up being really fragile.

I think I'd look at either not inheriting and finding a different way to share a bunch of style/logic (through style modules and mixins) or expose attributes/slots of the base template so that inherited components could customize it.